nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -42,7 +42,7 @@
   by setting up custom EDID files and forcing kernel/framebuffer modes.
 
 - [`services.displayManager.ly`](#opt-services.displayManager.ly.enable) is a new module for configuring the display manager [ly](https://github.com/fairyglade/ly),
-  a TUI-based replacement for SDDM and LightDM meant for window manager users.
+  a TUI-based replacement for SDDM and LightDM meant for window manager users.rl-24
```

Comment:
I don't think this change was intentional.

Reply:
done